app/test_main.py

```diff
@@ -1,3 +1,67 @@
-from app.main import get_human_age
+import pytest
 
-# write your code here
+from app.main import get_human_age, convert_to_human
+
+
+@pytest.mark.parametrize(
+    "testing_cat_age, testing_dog_age, expected_result",
+    [
+        pytest.param(
+            15, 15, [1, 1],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            24, 24, [2, 2],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            100, 100, [21, 17],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            14, 14, [0, 0],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="result_should_be_correct"
+        )
+    ]
+)
+def test_should_check_correct_data(
+        testing_cat_age: int,
```

Comment:
```suggestion
        cat_age: int,
```
I believe it's an extra prefix